README.md

```diff
@@ -111,16 +111,62 @@ If you installed globally, run
 prettier path/to/file.php --write
 ```
 
+## Editor integration
+
+Integration in the prettier plugin for your favorite editor might not be working yet, see see the related issues for [VS Code](https://github.com/prettier/prettier-vscode/issues/395), [Atom](https://github.com/prettier/prettier-atom/issues/395) and [Vim](https://github.com/prettier/vim-prettier/issues/119).
+
+For the moment, you can set up prettier to run on save like this:
+
+### Atom
+
+Install [save-autorun](https://atom.io/packages/save-autorun) and create a `.save.cson` file in your project with the following content:
+
+```cson
```

Comment:
cson?

Reply:
Yeah, the plugin seems to use cson instead of json (coffeescript object notation)